src/routes/(console)/+layout.ts

```diff
@@ -28,6 +29,20 @@ export const load: LayoutLoad = async ({ depends, parent }) => {
         preferences.organization ??
         (organizations.teams.length > 0 ? organizations.teams[0].$id : undefined);
 
+    // Load projects for the current organization if one is selected
+    let projects = null;
+    if (currentOrgId) {
+        try {
+            projects = await sdk.forConsole.projects.list([
+                Query.equal('teamId', currentOrgId),
+                Query.limit(1000) // Get all projects for organization
```

Comment:
this is a very heavy operation, payload wise. The returned data is very huge. We should definitely allow more queries on this endpoint in the backend and only fetch whats needed via `Query.select` later.